tests/qos/tunnel_qos_remap_base.py

```diff
@@ -95,8 +95,8 @@ def get_queue_counter(duthost, port, queue, clear_before_read=False):
     """
     if clear_before_read:
         duthost.shell("sonic-clear queuecounters")
-    # Wait a default interval (10 seconds)
-    time.sleep(10)
+        # Wait a default interval (10 seconds)
+        time.sleep(10)
```

Comment:
Please double check if the sleep is required. Even if we don't clear the counter, we still need to wait for a query interval to ensure queue counter is refreshed.

Reply:
After 10 sec delay in queue counter reading, pfc frames sending might actually had already stopped.
So bounce back packet might still send out, and queue counter increased accordingly.
And then caused flaky test faiure.
Temporarily disable the assert queue counter here until find a better solution, such as reading counter using sai thrift API